src/v0/sources/pipedream/transform.js

```diff
@@ -0,0 +1,35 @@
+const Message = require("../message");
+const { generateUUID } = require("../../util");
+const { callTypes } = require("./config");
+const { findUserIdOrAnonymousId } = require("./util");
+
+const buildTrackPayload = event => {
+  const message = new Message(`PIPEDREAM`);
+  message.setEventType("track");
+  message.setEventName("pipedream_source_event");
+  message.setProperty("properties", event);
+  return message;
+};
+
+const processEvent = event => {
+  const userIdorAnonymousId = findUserIdOrAnonymousId(event);
```

Comment:
We can try a better name here, like "identifier". @ItsSudip

Reply:
Yess, Will update it.